### PR TITLE
feat: add tts command line subcommand

### PR DESCRIPTION
**Pull request by jespino**
This PR adds the tts (Text to Speach) command to the localai binary.

This PR is related to the issue #816

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -274,6 +277,69 @@ For a list of compatible model, check out: https://localai.io/model-compatibilit
 					},
 				},
 			},
+			{
+				Name:  "tts",
+				Usage: "Convert text to speach",
```
Comment (mudler) [inline main.go:282]:
```suggestion
				Usage: "Convert text to speech",
```

Comment (jespino) [reply main.go:282]: :man_facepalming:

Comment (jespino) [reply main.go]: Done

**Review by mudler — APPROVED** [review]
looking good!

**Pull request merged**